api/dimensions.go

```diff
@@ -32,7 +32,7 @@ func (api *DatasetAPI) getDimensions(w http.ResponseWriter, r *http.Request, lim
 	logData := log.Data{"dataset_id": datasetID, "edition": edition, "version": version, "func": "getDimensions"}
 	var err error
 
-	versionNumber, err := models.ValidateVersionNumber(ctx, version)
+	versionNumber, err := models.ParseAndValidateVersionNumber(ctx, version)
```

Comment:
I do think that `ParseVersionNumber()` is a better name, I'd say validation is implied by parsing :)